docs/user-docs/latest/media-library/organizing-assets-with-folders.md

```diff
@@ -28,6 +28,10 @@ From this dedicated folder view, folders and assets can be managed, filtered, so
 
 To navigate back to the parent folder, one level up, use the **Back** button at the top of the interface.
 
+::: tip
+Breadcrumbs can also be used to navigate to a different folder: click on a folder name to directly jump to it or click on the 3 dots `...` and select a parent folder from the drop-down list.
```

Comment:
```suggestion
The breadcrumb navigation can also be used to navigate to a different folder: click on a folder name to directly jump to it or click on the 3 dots `...` and select a parent folder from the drop-down list.
```

Reply:
Thank you, Gustav :) I think @maevalienard wanted me to simply keep "breadcrumbs," but maybe it would make sense in this context 🤔 (though I'm not sure about "The […] navigation can be used to navigate")

Reply:
I feel like it's a bit repetitive indeed, but regarding the way to call that, I think "The breadcrumb navigation" is still pretty straight forward so all good on my side actually :D